src/main/java/frc/robot/subsystems/SimulationSubsystem.java

```diff
@@ -46,8 +53,12 @@ public void simulationPeriodic() {
     //Update simYaw distance
     simYaw += chassisRotationSpeed *  Robot.kDefaultPeriod;
 
+    //Update simPitch
+    //simPitch -= 
```

Comment:
You could use the swerveSubsystem.getPose() to determine the location in simulation and what the pitch will be in our first-order iteration

Reply:
was released with draft, is now updated